### PR TITLE
SEO-3147 Angular Desc too long short

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,

Please review the changes for Angular Desc too long short.

Regards,
Yvone.

### CODE REVIEW

File: angular/DateTimePicker/Getting-Started.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 title: Getting Started | DateTimePicker | Angular | Syncfusion
-description: getting started
+description: Learn here more about Getting Started with Syncfusion Essential Angular DateTimePicker Control, its elements, and more.
 platform: Angular
 control: DateTimePicker
 documentation: ug
 ---
 
-# Getting Started
+# Getting Started with Essential Angular DateTimePicker
```
Comment (Aishwarya-Ganesan) [inline angular/DateTimePicker/Getting-Started.md:10]: Remove Essential word.

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented.

File: angular/DateTimePicker/Getting-Started.md (hunk 1 above)
Comment (Yvone-Atieno) [reply angular/DateTimePicker/Getting-Started.md:10]: removed

File: angular/ReportViewer/Getting-Started.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-description: ReportViewer
+description: Learn here more about Getting started with Syncfusion Essential Angular ReportViewer Control, its elements, and more.
 platform: Angular
 control: ReportViewer
 documentation: ug
 keywords: ejReportViewer, ReportViewer, js ReportViewer
 ---
 
-# Getting Started
+# Getting Started with Essential Angular ReportViewer
```
Comment (gowthamanthavasiyappan) [inline angular/ReportViewer/Getting-Started.md:10]: Remove `Essential` word.

File: angular/Grid/Overview.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-description: Overview of Grid control
+description: Learn here more about Overview support in Syncfusion Essential Angular Grid Control, its elements, and more.
 platform: Angular
 control: Grid
 documentation: ug
 keywords: ejGrid, Grid, Grid widget
 api: /api/Angular/grid
 ---
-# Overview
+# Overview in Essential Angular Grid
```
Comment (gowthamanthavasiyappan) [inline angular/Grid/Overview.md:10]: Remove Essential word.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]

**Comment by gowthamanthavasiyappan** [comment]
Commented

File: angular/Grid/Overview.md (hunk 3 above)
Comment (Yvone-Atieno) [reply angular/Grid/Overview.md]: changed

File: angular/ReportViewer/Getting-Started.md (hunk 2 above)
Comment (Yvone-Atieno) [reply angular/ReportViewer/Getting-Started.md]: changed

**Review by gowthamanthavasiyappan — APPROVED** [review]

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Changes are fine.

**Pull request merged**